ng/src/web/pages/filters/dialog.js

```diff
@@ -23,89 +24,114 @@
 
 import React from 'react';
 
-import  _ from 'gmp/locale.js';
+import _ from 'gmp/locale.js';
 
 import Layout from '../../components/layout/layout.js';
 
 import PropTypes from '../../utils/proptypes.js';
 
-import withDialog from '../../components/dialog/withDialog.js';
+import SaveDialog from '../../components/dialog/savedialog.js';
 
 import FormGroup from '../../components/form/formgroup.js';
 import TextField from '../../components/form/textfield.js';
 import Select from '../../components/form/select.js';
 
 const FilterDialog = ({
-    comment,
-    name,
+    filter,
     term,
+    title = _('New Filter'),
     type,
     types,
-    onValueChange,
+    visible,
+    onClose,
+    onSave,
   }) => {
+  const defaults = {name: _('Unnamed'), term: ''};
+
   return (
-    <Layout flex="column">
-      <FormGroup title={_('Name')}>
-        <TextField
-          name="name"
-          grow="1"
-          value={name}
-          size="30"
-          onChange={onValueChange}
-          maxLength="80"/>
-      </FormGroup>
+    <SaveDialog
+      visible={visible}
+      title={title}
+      onClose={onClose}
+      onSave={onSave}
+      initialData={{
+        ...defaults,
+        ...filter,
+        term,
+        type,
+      }}
+    >
+      {({
+        data: state,
+        onValueChange,
+      }) => {
+        return (
+          <Layout flex="column">
+            <FormGroup title={_('Name')}>
+              <TextField
+                name="name"
+                grow="1"
+                value={state.name}
+                size="30"
+                onChange={onValueChange}
+                maxLength="80"/>
+            </FormGroup>
 
-      <FormGroup title={_('Comment')}>
-        <TextField
-          name="comment"
-          grow="1"
-          value={comment}
-          size="30"
-          maxLength="400"
-          onChange={onValueChange}/>
-      </FormGroup>
+            <FormGroup title={_('Comment')}>
+              <TextField
+                name="comment"
+                grow="1"
+                value={state.comment}
+                size="30"
+                maxLength="400"
+                onChange={onValueChange}/>
+            </FormGroup>
 
-      <FormGroup title={_('Term')}>
-        <TextField
-          name="term"
-          grow="1"
-          value={term}
-          size="30"
-          onChange={onValueChange}/>
-      </FormGroup>
+            <FormGroup title={_('Term')}>
+              <TextField
+                name="term"
+                grow="1"
+                value={state.term}
+                size="30"
+                onChange={onValueChange}/>
+            </FormGroup>
```

Comment:
Same as above. Maybe here are tabs included.

Reply:
I deleted all indentations completely and inserted new ones. Atom/Git doesn't recognize any changes. I don't know, where the indentations on GitHub are from.